configs/alcf/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta

```diff
@@ -115,35 +109,14 @@ endYear = 9999
 ## options related to ocean observations with which the results will be compared
 
 # directory where ocean observations are stored
-baseDirectory = /projects/ccsm/acme/diagnostics_data/observations/Ocean
-sstSubdirectory = SST
-sssSubdirectory = SSS
-mldSubdirectory = MLD
-ninoSubdirectory = Nino
-mhtSubdirectory = MHT
-meltSubdirectory = Melt
-soseSubdirectory = SOSE
-
-[oceanPreprocessedReference]
-## options related to preprocessed ocean reference run with which the results
-## will be compared (e.g. a POP, CESM or ACME v0 run)
-
-# directory where ocean reference simulation results are stored
-baseDirectory = /projects/ccsm/acme/diagnostics_data/reference_runs/ACMEv0_lowres/B1850C5_ne30_v0.4/ocn/postprocessing
+baseDirectory = /lus/theta-fs0/projects/EnergyClimate_2/diagnostics/observations/Ocean
```

Comment:
same change above

Reply:
no wait, this is actually the v0 stuff. I think we said we should remove it all together, right?

Reply:
never mind my previous comment..